contrib/scaling/Makefile

```diff
@@ -11,7 +11,7 @@ mpi_no_op: mpi_no_op.c
 	mpicc -o mpi_no_op mpi_no_op.c
 
 mpi_memprobe: mpi_memprobe.c
-	mpicc -o mpi_memprobe mpi_memprobe.c -lopen-pal -lopen-rte
+	mpicc -o mpi_memprobe mpi_memprobe.c -lopen-pal
```

Comment:
This is really a pointless change.  The test itself `#include`s a bunch of orte headers, so this file isn't going to compile in the ORTE-less world anyway.

Reply:
Is there a reason to keep it around though?